Use v1 of numeral

MarcusNoble opened the pull request:
Numeral just update to v2 introducing breaking changes. This change pins it to version 1.

MarcusNoble committed:
Use v1 of numeral

solidgoldpig commented:
Erm, shouldn't this be numeral that you've pinned to ~1 rather than handlebars?

Also, I note that I completely forgot to add lodash and must have had it installed globally back in the day when I wrote the tests. (If you fancy adding that dependency as well as fixing the numeral version fix ;)

MarcusNoble commented:
Doh! Clumsy! I'll update! I'll also add lodash.

MarcusNoble committed:
Pinned correct dep

MarcusNoble commented:
Fixed and added lodash. The GH diff makes it look like I've changed the whole file. https://github.com/solidgoldpig/handlebars.numeral/pull/1/files?w=1 Makes things much clearer.
